libtcmu_log.c

```diff
@@ -209,6 +209,34 @@ static void log_output(int pri, const char *msg, bool bypass)
 	}
 }
 
+struct log_buffer {
```

Comment:
Let's add a "early" prefix or something similar to avoid confusion with the other definitions that are similar. Also try to name them similar to the normal ones, so we would have

struct early_log_buf instead of log_buffer
tcmu_early_logbuf instead of log_tmpbuf
tcmu_early_logbuf_lock instead of log_tmpbuf_lock

Reply:
Will fix it.